components/function-controller/design/2022-10-v1alpha2-templating.md

```diff
@@ -0,0 +1,67 @@
+# Function templating 
```

Comment:
Could we move this document to `adr` directory instead of `design`. ADR (sometimes ADL) is well known short. 

More about ADR: https://github.com/joelparkerhenderson/architecture-decision-record